android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -44,10 +46,11 @@ class VideoPlaybackService : MediaSessionService() {
 
     // Player Registry
 
-    fun registerPlayer(player: ExoPlayer) {
+    fun registerPlayer(player: ExoPlayer, from: Class<android.app.Activity>) {
         if (mediaSessionsList.containsKey(player)) {
             return
         }
+        startedFrom = from
```

Comment:
Can you please set startedFrom = null in unregisterPlayer ?
maybe you can find a better variable name, something like sourceActivity

Reply:
Done